sources/packages/web/src/views/student/StudentDashboard.vue

```diff
@@ -1,21 +1,13 @@
 <template>
-  <div class="p-m-4">
-    <h1>Student Dashboard</h1>
-  </div>
-  <Card class="p-m-4">
-    <template #title>
-      Welcome {{ user.givenNames }}, let's get started
-    </template>
-    <template #content>
-      Student Dashboard Work in Progress
-    </template>
-  </Card>
+  <formio :formName="'studentDashboard'"></formio>
```

Comment:
the path in the formio is studentdashboard, please change it with the proper case